File: Baekjoon/step by step/08_math_1/2903.py
# 2903: 중앙 이동 알고리즘

N = int(input())

current_point = 2

for _ in range(N):
  current_point += current_point-1

print(current_point ** 2)

# 2

# 3(2 + (2-1)) . . .

# 5(3 + (3-1)) .. .. .

# 9(5 + (5-1))' " ' " ' " ' " '

# 17(9 + (9-1))' ' ' ' ' ' ' ' '

# 33(17 + (17-1))

File: Baekjoon/step by step/08_math_1/2903.cpp
// 2903: 중앙 이동 알고리즘

#include <iostream>

int main() {
  int N, current_point=2;
  std::cin >> N;

  while (N--) {
    current_point += current_point-1;
  }

  std::cout << current_point * current_point;

  return 0;
}